### ISSUE TITLE
@requestExample and @responseExample does not seem to work with spring

### ISSUE THREAD

**Issue by Amaury-Courjault** [stoicflame/enunciate#661]
Hi, 
I am trying to generate my custom request/response examples (with the new 2.9.1 version) for my spring project and annotation does not seem to be applied to the documentation example section of the request. 
Here are the configuration files I am using :

My rest controller :
```
@RestController
@RequestMapping("/test")
public class TestRestController {

    /**
     * @requestExample application/json { "name" : "TEST" }
     */
    @RequestMapping(value = "/doc", method = RequestMethod.POST, consumes = MediaType.APPLICATION_JSON_VALUE, produces = MediaType.APPLICATION_JSON_VALUE)
    Map<String, String> testDoc(@RequestBody Map<String, String> testBody) {
        return testBody;
    }
}
```

My enunciate.xml :
```
<?xml version="1.0" encoding="UTF-8"?>
<enunciate xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://enunciate.webcohesion.com/schemas/enunciate-2.9.0.xsd">

    <modules>
        <swagger disabled="true"/>
        <gwt-json-overlay disabled="true"/>
        <java-xml-client disabled="true"/>
        <java-json-client disabled="true"/>
        <c-xml-client disabled="true"/>
        <csharp-xml-client disabled="true"/>
        <javascript-client disabled="true"/>
        <obj-c-xml-client disabled="true"/>
        <php-xml-client disabled="true"/>
        <php-json-client disabled="true"/>
        <ruby-json-client disabled="true"/>
        <idl disabled="true"/>
    </modules>

</enunciate>
```

Generated doc result : 
![enunciate](https://cloud.githubusercontent.com/assets/5901407/26239108/077fad48-3c7c-11e7-95ff-1312b34c1ce9.png)

Maybe I am not using the annotation in the right way...

**Comment by stoicflame** [comment]
Bug confirmed. Fixed at d74bc76.

**Comment by stoicflame** [comment]
[Enunciate 2.10 is now available](https://github.com/stoicflame/enunciate/releases/tag/v2.10.0).
